app/code/Magento/InventoryInStorePickupAdminUi/view/adminhtml/ui_component/inventory_source_form.xml

```diff
@@ -48,6 +48,18 @@
             </settings>
         </field>
     </fieldset>
+    <fieldset name="contact_info">
+        <field name="phone" component="Magento_InventoryInStorePickupAdminUi/js/form/element/conditional-required">
+            <settings>
+                <imports>
+                    <link name="required">${ $.provider }:data.general.extension_attributes.is_pickup_location_active</link>
+                </imports>
+                <validation>
+                    <rule name="validate-phoneLax" xsi:type="boolean">true</rule>
```

Comment:
I don't think we should validate the phone number. At least in the admin panel.

There are too many phone formats in the world, I can see more possible issues that profit there.

https://en.wikipedia.org/wiki/National_conventions_for_writing_telephone_numbers

![image](https://user-images.githubusercontent.com/26086561/68681624-62c66f00-056c-11ea-854e-0067b5710268.png)

Reply:
Agree.
Removed.